### ISSUE TITLE
RealmList is not saved as part of model

### ISSUE THREAD

**Issue by tarasmorskyi-mediamonks** [realm/realm-kotlin#334]
If I have a model with RealmList in it the list is not getting copied into realm as any other field does. Example of used model:
class SampleEntity : RealmObject {
   @PrimaryKey
    var productId: String = ""
    var price: Double = 0.0
    var children: RealmList<SampleChildEntity> = RealmList()
}

**Comment by cmelchior** [comment]
Thanks @tarasmorskyi-mediamonks 

Exactlly what do you mean by being copied into Realm? Do you have a full example you can post?

**Comment by tarasmorskyi-mediamonks** [comment]
unfortunately I cannot share the real peace of code but this will represent the idea:
![Screenshot 2021-07-02 at 14 47 21](https://user-images.githubusercontent.com/50660533/124276942-bc661d80-db44-11eb-8b27-de2dfa8a5df9.png)
![Screenshot 2021-07-02 at 14 49 01](https://user-images.githubusercontent.com/50660533/124276949-bf610e00-db44-11eb-8955-084b959cbfdb.png)

**Comment by nhachicha** [comment]
Hi @tarasmorskyi-mediamonks I can reproduce your issue. We'll look into it, in the meantime you can use the following workaround 

```Kotlin
realm.writeBlocking {
            copyToRealm(sampleEntity).run {
                copyToRealm(SampleChildEntity().apply {
                    childId = "1"
                    title = "one"
                }).also { children.add(it) }

                copyToRealm(SampleChildEntity().apply {
                    childId = "2"
                    title = "two"
                }).also { children.add(it) }
            }

        }
```

**Comment by edualonso** [comment]
A fix has been shipped in v0.3.2. Please reopen the issue if the problem still persists.
